src/main/java/org/opentestsystem/ap/imrt/iis/message/ItemUpdateMessageListener.java

```
@@ -0,0 +1,41 @@
+package org.opentestsystem.ap.imrt.iis.message;
+
+import org.opentestsystem.ap.imrt.common.service.OperationalEventService;
+import org.opentestsystem.ap.imrt.iis.service.ItemUpdateNotificationHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * Handles processing of item update message listener
+ */
+@Component
+public class ItemUpdateMessageListener {
```

I pulled the message handling out to allow for some additional logging and allowing the ItemUpdateNotificationHandler to focus on processing the update message and not have to deal with error handling.  The handler now will let Exceptions bubble and will be logged here.